ui/src/app/pages/runtimes-list.tsx

```diff
@@ -81,12 +81,22 @@ const css =
       .join(',\n')
   ) +
   ` {
-         display: none !important
-        }
-      
-       div[style*="z-index: 2"]:has(>div>svg) {
-          display: none !important
-        }`;
+        display: none !important
+    }
+    
+    div[style*="z-index: 2"]:has(>div>svg) {
```

Comment:
Could you comment on what this does @NehaBroad @dmohs ?  I saw that it was introduced in https://github.com/all-of-us/workbench/pull/7463

Reply:
Yes, should have added comment. I believe it was a warning message on the cloud environments page.

Reply:
OK.  How does the comment I added look?